json/src/lib.rs

```diff
@@ -2105,6 +2105,13 @@ pub struct MasternodeStatus {
 
 // --------------------------- Quorum -------------------------------
 
+#[serde(untagged)]
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub enum QuorumType{
```

Comment:
This should be enum on the different quorum types with methods to transform into a u8 and to describe as a String.

Reply:
Done